packages/pillarbox-debug-panel/scss/pillarbox-debug-panel.scss

```diff
@@ -0,0 +1,44 @@
+.pbw-debug-panel {
+  position: absolute;
+  bottom: 6em;
+  left: 2.1em;
+  width: 30%;
+  padding: 0.8em;
+  color: white;
+  font-size: 0.8em;
```

Comment:
When using the default video-js css theme, the font size is a bit too small, I would suggest keeping the default font size.